README.md

```diff
@@ -1,29 +1,26 @@
 # Airship Plugin
 
-Preview of the Airship Flutter plugin.
+The Airship Flutter plugin allows using Airship's native iOS and Android APIs with Flutter apps written in Dart.
 
-This plugin is currently in development or published. APIs will most likely change before the first
-major release.
+## Usage
 
-## Status
+1. Add the airship dependency to your package's pubspec.yaml file:
 
-Feature Status:
-- [x] Tags
-- [x] Notifications
-- [x] Deep links
-- [x] Inbox
-- [x] Named User
-- [x] Push Events
-- [ ] Custom events
```

Comment:
are these non-checked items finished? if not do we have tickets for them?

Reply:
They are finished